superset/utils/core.py

```diff
@@ -860,6 +861,7 @@ def get_filter_key(f):
                 existing_filters[get_filter_key(existing)] = existing["comparator"]
 
         for filtr in form_data["extra_filters"]:
+            filtr["isExtra"] = True
             # Pull out time filters/options and merge into form data
             if date_options.get(filtr["col"]):
                 if filtr.get("val"):
```

Comment:
nit: `extra_filter` or `filter_`

Reply:
I agree `filter_ ` is slightly better but being lazy and mergin' as is.
![ship](https://user-images.githubusercontent.com/487433/77393532-74a9f100-6d5a-11ea-919d-9bc6d9366fac.gif)